synapse/models/person.py

```diff
@@ -25,6 +25,8 @@ def getModelDefs(self):
                     'ex': 'robert grey'
                 }),
                 ('ps:person', ('guid', {}), {
+                    'interfaces': ('phys:object',),
+                    'template': {'phys:object': 'person'},
```

Comment:
do we need to deconflict w/ps:vitals (which has :height and :weight)?

Reply:
Yep, good catch!